test/runtests.jl

```diff
@@ -35,6 +36,12 @@ end
         t_stop = 6 * 3600,
     )
     nothing
+
+    # Test running from parsed namelist
+    namelist_path = "Output.Bomex.01/namelist_Bomex.in"
+    read_namelist = JSON.parsefile(namelist_path)
```

Comment:
```suggestion
    namelist_filename = "Output.Bomex.01/namelist_Bomex.in"
    read_namelist = JSON.parsefile(namelist_filename)
```
I think `path` usually refers to folders (e.g., `mkpath`, `ispath`)